### PR TITLE
feat: Add activities and notifications for adding and removing space members

### LINKED ISSUE #671: Operations that add and remove members from and to spaces not recording activity
The following operations are not creating an activity record:

- lib/operately/operations/group_members_adding.ex
- lib/operately/operations/group_member_removing.ex

### PR DESCRIPTION
**Pull request by Rockyy174**
I've handled the issue described in https://github.com/operately/operately/issues/671.

Now adding members to a space or removing them from it creates an activity record. These records are going to be displayed in the Feed.
Also, members who are added to a space should receive an email notification.

### CODE REVIEW

File: lib/operately_email/templates/space_members_added.text.eex [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<%= short_name(@author) %> added yout to the <%= @space.name %> space.
```
Comment (shiroyasha) [inline lib/operately_email/templates/space_members_added.text.eex:1]: Typo `yout` -> `you`

**Comment by shiroyasha** [comment]
Looks good 👍 Apart from the small typo, everything else is ready for 🚢

File: lib/operately_email/templates/space_members_added.text.eex (hunk 1 above)
Comment (Rockyy174) [reply lib/operately_email/templates/space_members_added.text.eex:1]: 😅

**Comment by shiroyasha** [comment]
It seems that the test failed, or is it flaky?

**Comment by Rockyy174** [comment]
I think it's flaky because it had passed before I fixed the typo. But is's failed about 3 times already 😬

**Pull request merged**